data/material-model/entropy-table/opxtable/README.md

```diff
@@ -0,0 +1,7 @@
+This data is example data produced by HeFESTo [1,2] for orthopyroxene under
+pressure-entropy conditions that occur in the Earth's mantle. The data itself
+is only useful for benchmarking the method as done in
+benchmarks/entropy\_adiabat. Its resolution is too coarse for realistic models.
+
```

Comment:
Can you explain what the different files are?

Reply:
Done.

Reply:
I don't see where you did this. I was looking for a short overview of the different datafiles:  data/material-model/entropy-table/opxtable/opxtable_s.aspect (which is the entropy-pressure table, I think that is clear), data/material-model/entropy-table/opxtable/opxtable_T.ad1600.txt (which it seems is the HeFESTo adiabat you can use for checking the test results), data/material-model/entropy-table/opxtable/opxtable_s.ad1600 and data/material-model/entropy-table/opxtable/opxtable_s.ad1600.txt (here, I don't really understand why you have 2. Is one the HeFESTo output? Is there a script to convert one to the other? Where is that used in the model? Why does this adiabat have two files, but the temperature adiabat has only one?)